utils/doclint/generateDotnetApi.js

```diff
@@ -199,6 +200,11 @@ const customTypeNames = new Map([
         out.push(`\t${escapedName},`);
       });
     }, enumsDir));
+  
+  if (process.argv[3] !== "--skip") {
```

Comment:
I'm sure there's a better way to do this, but this solves the problem _for now_ and lets us skip the formatting on machines that may not have `dotnet` installed.

Reply:
should it be `--skip-format` ?